docs/feature-flagging/qa-flags.md

```diff
@@ -0,0 +1,29 @@
+# QA Flags
+
+## Testing Environments
+
+Eppo supports separation of feature flags and their allocations between environments. Create an Eppo environment specifically for Staging or QA following this [documentation](/sdks/api-keys). Environments are separated by [SDK keys](/sdks/api-keys), and no unique key will be shared across Eppo Environments. Once an Environment is created, create an SDK for that Environment.
+
+![Screenshot of SDK key setup](/img/feature-flagging/qa-flag-sdk-key.png)
+
+After a flag has been created, click **Switch Environment** on the flag page and select the testing Environment the flag should run in.
+
+![Screenshot of list of Environments](/img/feature-flagging/qa-flag-environments.png)
+
+Click **Add Allocation** on the flag and serve the treatment or the experience to be tested to 100%.
+
+Set your flag to **On** to have it run in the selected environment.
+
+![Screenshot of allocation for Staging Environment](/img/feature-flagging/qa-flag-staging-allocation.png)
+
+## Testing in Production
```

Comment:
You may also want to mention that the user can copy their allocation from testing to production

Reply:
included that, great suggestion